docs/tutorials/training_agents/deep_rl_tutorial.py

```diff
@@ -0,0 +1,1095 @@
+#
+# Also, since agents are trained with different seeds, the average and standard deviation is what is plotted.
+#
+# The function ``visualize_performance`` does the plotting, while the function ``aggregate_results``
+# computes mean&stddev of the different agents results and ``preprocess_results`` does the combining of the metrics.
+# Furthermore, the ``collect_video_frames`` is used to play a number of episodes and return the frames, so that
+# ``create_gif`` can be used to create a gif of these frames.
+#
+def aggregate_results(lst: List[np.ndarray]) -> (np.ndarray, np.ndarray):
+    """Aggregate a list of arrays to compute their mean and stddev."""
+    average = np.mean(lst, axis=0).round(2)
+    stddev = np.std(lst, axis=0).round(2)
+
+    return average, stddev
+
+
+def preprocess_results(
+    results: List[MetricsLogger],
+) -> List[Tuple[np.ndarray, np.ndarray]]:
+    """Combine data for various metrics and aggregate them across agents. Return the processed data."""
+    stats = [
+        [res_buffer.episode_returns for res_buffer in results],
+        [res_buffer.episode_lengths for res_buffer in results],
+        [res_buffer.episode_action_values for res_buffer in results],
+        [res_buffer.losses for res_buffer in results],
+        [res_buffer.policy_entropy for res_buffer in results],
+        [res_buffer.evaluation_returns for res_buffer in results],
+    ]
+
+    aggregated_data = [aggregate_results(lst) for lst in stats]
+
+    return aggregated_data
+
+
+def visualize_performance(
+    processed_data: List[Tuple[np.ndarray, np.ndarray]],
+    baseline_returns: np.ndarray,
+    params: Hyperparameters,
+):
+    """
+    Visualize the aggregated metrics collected by the agents.
+
+    Args:
+        processed_data: A list containing tuples of (mean, stddev) for each metric.
+        baseline_returns: Array of the random baseline episodic returns.
+        params: Hyperparameters namedtuple.
+
+    Returns:
+
+    """
+    plt.style.use("seaborn")
+    x = np.linspace(
+        params.record_statistics_fraction * params.training_steps,
+        params.training_steps,
+        int(1 / params.record_statistics_fraction),
+    )
+    color = "royalblue"
+    y_labels = [
+        "Return",
+        "Episode Length",
+        "Predicted action-value",
+        "Loss",
+        "Entropy",
+    ]
+    titles = [
+        "Aggregated agents returns vs baseline",
+        "Aggregated episode lengths",
+        "Aggregated action-value per episode",
+        "Aggregated training losses",
+        "Aggregated policy entropy",
+    ]
+
+    fig, axes = plt.subplots(nrows=3, ncols=2, figsize=(14, 21))
+
+    eval_axes = axes.flatten()[-1]
+    axes = axes.flatten()[:-1]
+
+    for i, ax in enumerate(axes):
+        mean, std = processed_data[i]
+        ax.plot(x, mean, color=color, label="mean")
+        ax.fill_between(
+            x=x, y1=mean - std, y2=mean + std, label="stddev", alpha=0.2, color="tomato"
```

Comment:
Since when is "tomato" a color? lol

Does `matplotlib` not complain???

Reply:
:D it is a beautiful one too. See [List of named colors](https://matplotlib.org/stable/gallery/color/named_colors.html)